Components/Alert/Alert.css

```diff
@@ -0,0 +1,65 @@
+.alert-container{
+    padding: 2rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.alert-item{
+    padding: 2rem 4rem;
+}
+
+.success-alert{
+    background-color: var(--success-alert);
+    margin: var(--alert-margin);
+    border-radius: var(--rem-border-radius);
```

Comment:
`border-radius` can be added in `.alert-item` , since it is common for all alert items.

Reply:
done! thanks for notifying